data/tr2/ship/cfg/TR2X_gameflow.json5

```diff
@@ -340,6 +340,8 @@
                 {"type": "set_lara_start_anim", "anim": 9},
                 {"type": "remove_weapons"},
                 {"type": "remove_ammo"},
+                {"type": "remove_flares"},
+                {"type": "remove_medipacks"},
```

Comment:
Worth adding a changelog/readme entry for this separation? Nice feature imo.

Reply:
IMO not really as we're already flipped the world upside down with the shift to JSON, but it does mandate a documentation change. I'll update